src/main/java/org/generationcp/middleware/dao/GermplasmDAO.java

```diff
@@ -757,7 +757,7 @@ public long countGroupRelatives(final Integer gid) {
 	public PedigreeDTO getPedigree(final Integer germplasmDbId, final String notation, final Boolean includeSiblings) {
```

Comment:
You can remove `GermplasmDataManager#getPedigree` and just call this `GermplasmDAO#getPedigree` directly from `GermplasmServiceImpl`